website/src/components/header.js

```diff
@@ -46,6 +46,9 @@ export default class Header extends Component {
             <a href={FRAMEWORK_GITHUB_URL}>
               Github<i className="icon icon-github" />
             </a>
+            <a href="https://github.com/mapbox/mapbox-gl-js">
```

Comment:
Can you remove this? deck.gl is Mapbox compatible, but it does not have to be used with Mapbox. We do link to the Mapbox repo in http://uber.github.io/react-map-gl/